src/make-scm-group-role.js

```diff
@@ -0,0 +1,46 @@
+import editRole from './util/edit-role';
+import {getProjects, hgmoPath} from './util/projects';
+
+module.exports.setup = (program) => {
+  return program
+    .command('make-scm-group-role <group>')
+    .option('-n, --noop', 'Don\'t change roles, just show difference')
+    .description('create or update a mozilla-group:scm_foo role, based on production-branches.json');
+};
+
+module.exports.run = async (group, options) => {
+  var taskcluster = require('taskcluster-client');
```

Comment:
Is there a reason why var is used here over let? Should both have the same scoping in this context but I noticed it was inconsistent with the new admin scripts being add, such as in src/check-repo-levels.js